data/core/macros/abilities.cfg

```diff
@@ -198,9 +198,9 @@ This ability will not cure an affected unit of poison, however, only delay its e
         cumulative=no
         name= _ "leadership"
         female_name= _ "female^leadership"
-        description= _ "This unit can lead your own units that are next to it, making them fight better.
+        description= _ "This unit can lead your troops in battle.
```

Comment:
what about "adjacent troops"? I think an indication what the units are only one hex away is needed to avoid confusion.

Reply:
It's described in the next line, "adjacent troops"